src/EditorFeatures/Test2/IntelliSense/IntellisenseQuickInfoBuilderTests.vb

```diff
@@ -665,5 +675,121 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             Assert.Equal(expected.ClassificationTypeName, actual.ClassificationTypeName)
             Assert.Equal(expected.Text, actual.Text)
         End Sub
+
+        Private Shared Function ContainerToString(element As Object) As String
+            Dim result = New StringBuilder
+            ContainerToString(element, "", result)
+            Return result.ToString()
+        End Function
+
+        Private Shared Sub ContainerToString(element As Object, indent As String, result As StringBuilder)
+            result.Append($"{indent}New {element.GetType().Name}(")
```

Comment:
nit: Can you put a blank line after this to help separate it out? It took me a bit to find where the `New <element name>(` was coming from